runtime-config.md

```diff
@@ -50,3 +50,15 @@ Only [mounts from the portable config](config.md#mount-points) will be mounted.
 ```
 
 See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [SetVolumeMountPoint](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365561(v=vs.85).aspx) in Windows.
+
+*Example (Hypervisor)*
+
+```json
+"mounts": {
+    "data": {
+        "type": "ext3",
+        "source": "qcow2:/volumes/data.qcow2",
```

Comment:
I'm really not sure this is needed. It seems too much like a runtime specific item.